modules/dxp/apps/portal-workflow/portal-workflow-kaleo-designer-web/src/main/resources/META-INF/resources/designer/js/definition-builder/diagram-builder/components/sidebar/sections/notifications/NotificationsInfo.js

```diff
@@ -119,19 +119,27 @@ const NotificationsInfo = ({
 	const [notificationName, setNotificationName] = useState(
 		selectedItem.data.notifications?.name?.[notificationIndex] || ''
 	);
+
 	const [notificationTypeEmail, setNotificationTypeEmail] = useState(
-		selectedItem.data.notifications?.notificationTypes?.[
+		selectedItem.data.notifications?.notificationTypes[notificationIndex][
 			notificationIndex
-		]?.includes('email') || false
+		]?.notificationType === 'email' || false
 	);
+
 	const [
 		notificationTypeUserNotification,
 		setNotificationTypeUserNotification,
 	] = useState(
-		selectedItem.data.notifications?.notificationTypes?.[
+		selectedItem.data.notifications?.notificationTypes[notificationIndex][
```

Comment:
Same as above. Try using array `includes` method to check if there is an object inside the array with a value of "user-notification" for the `notificationType` property. This check should be enough, making the `||` operator unnecessary.